spec.html

```diff
@@ -194,6 +156,7 @@
   title: ECMAScript<sup>&reg;</sup> 2026 Language&nbsp;Specification
   shortname: ECMA-262
   status: draft
+  version: 17<sup>th</sup> edition
```

Comment:
This actually causes the document to render as "17th edition" instead of "Draft", which we definitely don't want for the living spec. I'll push up a commit removing it and then it should be good to go.

<img width="592" height="175" alt="Screen Shot 2025-08-25 at 6 45 23 PM" src="https://github.com/user-attachments/assets/9f264fb8-6625-42a0-ad74-aff2cd9676dd" />